src/modules/data/entities/aggregate/common/metagame.territory.result.embed.ts

```diff
@@ -22,6 +23,11 @@ export default class MetagameTerritoryResultEmbed {
     })
     cutoff: number;
 
+    @Column({
+        type: 'number',
+    })
```

Comment:
Not sure if this works but the other enums are written like
```
    @Column({
        type: 'enum',
        enum: factionArray,
    })
```

Reply:
3b410f2269ebc20d461ee4022d7e81f715fc642b